src/istgt_lu.c

```diff
@@ -1737,11 +1758,51 @@ istgt_lu_add_unit(ISTGT_Ptr istgt, CF_SECTION *sp)
 	ISTGT_TRACELOG(ISTGT_TRACE_DEBUG, "ConsistencyFactor %d\n",
 	    lu->consistency_factor);
 
+	if (lu->replication_factor > lu->desired_replication_factor) {
+		ISTGT_ERRLOG("Invalid config ReplicationFactor is"
+		    " greater than Desired ReplicationFactor\n");
+		goto error_return;
+	}
 	if (lu->replication_factor <= 0 || lu->consistency_factor <= 0 ||
 		lu->replication_factor < lu->consistency_factor) {
 		ISTGT_ERRLOG("Invalid ReplicationFactor/ConsistencyFactor or their ratio\n");
 		goto error_return;
 	}
+	/* Read known replica details from conf file and maintain in memory structure*/
+	val = istgt_get_val(sp, "Replica");
+	if (val == NULL) {
+		TAILQ_INIT(&lu->known_replica);
```

Comment:
how about making it as known_replica_list

Reply:
updated to `trusty_replicas`